src/develop/pixelpipe_hb.c

```diff
@@ -275,8 +275,10 @@ void dt_dev_pixelpipe_create_nodes(dt_dev_pixelpipe_t *pipe, dt_develop_t *dev)
     piece->histogram_params.bins_count = 256;
     piece->histogram_stats.bins_count = 0;
     piece->histogram_stats.pixels = 0;
-    piece->colors = ((module->default_colorspace(module, pipe, NULL) == iop_cs_RAW) && 
-          (pipe->image.flags & DT_IMAGE_RAW)) ? 1 : 4;
+    piece->colors
```

Comment:
Just a comment, this is reformatting it is always better to have this in a separate commit. Why?

Because when I see a "minor reformatting" or "style fix" commit I can unplug my brain and just merge the changes. But for every line in a standard commit I read it and try to make sense of it. Thanks.

Reply:
Sorry about that, again clang-format. I think I'll stop using it. Do you want me to reverse-it?

Reply:
No, this is now merged.